src/server/pfs/pretty/pretty.go

```diff
@@ -116,7 +117,7 @@ Parent: {{.ParentCommit.ID}}{{end}}
 Started: {{prettyAgo .Started}}{{if .Finished}}
 Finished: {{prettyAgo .Finished}} {{end}}
 Size: {{prettySize .SizeBytes}}{{if .Provenance}}
-Provenance: {{range .Provenance}} {{.Repo.Name}}/{{.ID}} {{end}} {{end}}
+Provenance: {{range .Provenance}} {{.Commit.Repo.Name}}/{{.Branch.Name}}/{{.Commit.ID}} {{end}} {{end}}
```

Comment:
I think `repo/branch/commit` is going to read a little weirdly, since we sometimes use that to represent files. Could we do `repo/commit (branch)`, the `branch` feels like kind of auxilliary data to me since it's mostly going to be `master` under normal usage.

Reply:
what if we omitted the branch unless it was something besides master?

Reply:
:+1: that sounds even better.